storage/data-plane/src/main/java/com/microsoft/azure/storage/blob/Metadata.java

```diff
@@ -13,6 +13,8 @@
  */
 public final class Metadata extends HashMap<String, String> {
 
+    private static final long serialVersionUID = -6557244540575247796L;
```

Comment:
Can you please add a comment that explains this is an offshoot of extending HashMap, which implements Serializable? This magic number will otherwise be confusing to future developers.

Reply:
Will do.

Reply:
Done